support-frontend/assets/helpers/tracking/acquisitions.js

```diff
@@ -223,6 +225,8 @@ const getAbTests = (
 function derivePaymentApiAcquisitionData(
   referrerAcquisitionData: ReferrerAcquisitionData,
   nativeAbParticipations: Participations,
+  baseContributionAmount?: number,
```

Comment:
does `baseContributionAmount` need to be optional? Shouldn't we always have it at this point?

Reply:
nope!